Shared/RollingFileLogger.swift

```diff
@@ -55,6 +57,22 @@ open class RollingFileLogger: XCGLogger {
             deleteOldestLogWithPrefix(self.root)
         }
     }
+    
+    open func deleteAllLogs() {
+        if logDirectoryPath == nil {
+            return
+        }
+        
+        do {
+            let logFiles = try FileManager.default.contentsOfDirectoryAtPath(logDirectoryPath!, withFilenamePrefix: root)
+            for filename in logFiles {
+                try FileManager.default.removeItem(atPath: "\(logDirectoryPath!)/\(filename)")
+            }
+        } catch _ as NSError {
```

Comment:
do we need this `NSError` cast?

Reply:
Copy pasted from the other function in this class